src/features/page_action/instrument/index.js

```diff
@@ -3,13 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { InstrumentBase } from '../../utils/instrument-base'
-import { FEATURE_NAME } from '../constants'
+import { GenericEvents } from '../../generic_events'
 
-export class Instrument extends InstrumentBase {
-  static featureName = FEATURE_NAME
+/**
+ * @deprecated This feature has been replaced by Generic Events. Use/Import `GenericEvents` instead. This wrapper will be removed in a future release
+ */
+export class Instrument extends GenericEvents {
```

Comment:
Instead of exporting a class here, we could delete all the files under `features/page_action` except the `index.js` file and make the `index.js` file contents the below.

```
import { Instrument as GenericEvents } from '../generic_events/instrument/index'

/**
 * @deprecated This feature has been replaced by Generic Events. Use/Import `GenericEvents` instead. This wrapper will be removed in a future release
 */
const PageAction = GenericEvents

export { PageAction }
```

This will ensure customers get the deprecation notice without breaking their code. This should work since we only list the `features/<feature_name>/index.js` files in the `package.json` exports field.

Reply:
We intentionally did not do this (and all the other entry re-routing mention) as it is less straightforward to understand for future us to have all the magic re-routing, having the pageActions entry point live in the generic events feature etc. instead of just keeping the page_action instrument class and having it use the generic events.  We can revise this rerouting in another effort if deemed necessary